guidance/library/_json.py

```diff
@@ -55,12 +57,14 @@ class Keyword(str, Enum):
 IGNORED_KEYS = {
     "$schema",
     "$id",
+    "id",
```

Comment:
I believe that our problem with `id` (in the FHIR schema) is only at top level. I think that `id` is a perfectly fine name for a property on an object.

Reply:
`validate_json_node_keys` is only being called on dictionaries that represent full (sub)schemas, which doesn't include the dictionary specified by the `properties` key of a (sub)schema. Therefore the set of ignored keys should have no impact on what property names are valid -- worth checking if you have doubts :)

Reply:
I'd suggest adding a test with an object with an `id` property.

Reply:
Happy to do so :)